workflow/automated-checks.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Automated checks
+description: How we set up automated checks for our GitHub projects
+---
+
+Every official team project should be set up to [require a number of checks to pass](https://help.github.com/en/github/administering-a-repository/enabling-required-status-checks) before a pull-request can be merged.
+
+## Prefer GitHub workflows
+
+These check should be written as [GitHub workflows](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow) if possible. If GitHub workflows aren't fit-for-purpose, our next choice should be [Circle CI](https://circleci.com/), and _only for the specific checks_ that can't be configured as GitHub workflows.
```

Comment:
```suggestion
These check should be written as [GitHub workflows](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow) if possible. If GitHub workflows aren't fit for purpose, our next choice should be [Circle CI](https://circleci.com/), and _only for the specific checks_ that can't be configured as GitHub workflows.
```
Why the hyphens?

Reply:
I don't know if we have a policy on this. I'm not the only one who would hyphenate this:
https://www.yourdictionary.com/fit-for-purpose
https://oilpatchwriting.wordpress.com/2012/04/13/multiple-hyphens/

I don't mind removing them, but in the absence of a specific copywriting policy, this feels like bike-shedding (note the hyphen :p).

Reply:
Ok, whatever you think.

Reply:
I discussed this with @pmahnke briefly. He correctly pointed out that Grammarly [explicitly recommends](https://www.grammarly.com/blog/hyphen/) using compound modifiers when possible. So I'm going to leave this as "fit-for-purpose", and if we did decide to add a policy later on I would suggest it err on the side of hyphens in cases like this, rather than not.

"Pull request" on the other hand is basically a noun coined by GitHub, and they don't hyphenate it. So I think you have a good point about that one.